frontend/src/pages/Dashboard.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+    <ContentLayout title="Dashboard">
+        dashboard text
+    </ContentLayout>
+</template>
+<script>
+    import ContentLayout from "../components/layout/ContentLayout";
+    export default {
+        name: 'Dashboard',
+        components: { ContentLayout }
```

Comment:
I think it's still worth adding the component of adding a site in its absence
mixins: [isWebsite],